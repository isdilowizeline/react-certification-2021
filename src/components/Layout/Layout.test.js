import React from 'react';
import {render} from '@testing-library/react';
import AuthProvider from '../../providers/Auth';
import Layout from '.';

describe('Layout Components', () => {
    it('should take a snapshot', () => {
        const {asFragment} = render(
            <AuthProvider>
                <Layout/>
            </AuthProvider>
        );
        expect(
            asFragment(
                <AuthProvider>
                    <Layout/>
                </AuthProvider>
            )
        ).toMatchSnapshot();
    });
});
