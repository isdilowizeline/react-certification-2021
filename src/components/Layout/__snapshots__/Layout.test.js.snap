// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fortune Components should take a snapshot 1`] = `
<DocumentFragment>
  <span
    class="fortune"
  >
     
  </span>
</DocumentFragment>
`;

exports[`Layout Components should take a snapshot 1`] = `
<DocumentFragment>
  <main
    class="container"
  />
</DocumentFragment>
`;
